### PR TITLE
Unify copyright statement formats and text

### PR DESCRIPTION
**Pull request by ddsharpe**

### CODE REVIEW

**Review by rosemarymarano — COMMENTED** [review]
According to http://pdb.us.oracle.com/downloads/legal_notices.html, the copyright statement does not include the word Corporation after Oracle: 
Copyright © [year,] year, Oracle and/or its affiliates.

File: samples/configurations/jvm.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+# Copyright 2019, Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
```
Comment (mriccell) [inline samples/configurations/jvm.yml:2]: The license must be https://oss.oracle.com/licenses/upl

File: samples/configurations/qualified.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+# Copyright 2019, Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
```
Comment (mriccell) [inline samples/configurations/qualified.yml:2]: https

**Review by mriccell — CHANGES_REQUESTED** [review]

**Comment by mriccell** [comment]
Every where you have a Licensing statement the URL must change from http to https

File: samples/configurations/qualified.yml (hunk 2 above)
Comment (ddsharpe) [reply samples/configurations/qualified.yml]: done

**Review by mriccell — APPROVED** [review]

**Review by rosemarymarano — APPROVED** [review]
Looks like you got them all.

**Pull request merged**